### ISSUE TITLE
Ci/CD Action : AL1033 error - Support for external rulesets?

### ISSUE THREAD

**Issue by fvet** [microsoft/AL-Go#730]
I'm currently testing the migration of an Azure DevOps project to Github. The app did contain a ruleset that had an 'IncludedRulesets' pointing towards an externally hosted ruleset file. (Especially usefull to manage our company rules only once and re-use across all repo's / projects)

When running the Ci/CD action, following error is raised:

```

Compiling...
  .\alc.exe /project:"c:\sources\Core\App" /packagecachepath:"c:\sources\.packages" /out:"c:\sources\.output\Dynavision_Dynavision Core_22.0.8.3.app" /analyzer:C:\build\vsix\extension\bin\Analyzers\Microsoft.Dynamics.Nav.CodeCop.dll /analyzer:C:\build\vsix\extension\bin\Analyzers\Microsoft.Dynamics.Nav.AppSourceCop.dll /ruleset:c:\sources\app.ruleset.json /assemblyprobingpaths:"C:\Program Files\dotnet\shared","C:\ProgramData\BcContainerHelper\Extensions\bc6291077182\.netPackages\Service"
  Microsoft (R) AL Compiler version 11.8.13.18663
  Copyright (C) Microsoft Corporation. All rights reserved
  
  Error: AL1033 An error occurred while loading the included rule set file c:\sources\app.ruleset.json - It was not possible to load the ruleset with location 'https://xxx.blob.core.windows.net/development/global.ruleset.json' because external rulesets are not allowed.
  App generation failed with exit code 1
  Compile-AppInBcContainer Telemetry Correlation Id: e9a96538-83b4-4143-b3ba-45e4361a4778
```

Am I missing a setting on Github / AL-Go or are external rulesets not supported at all? 

Apparently bccontainerhelper does support a 'EnableExternalRulesets' parameter to pass to the alc.exe parameters, but not sure if supported by AL-GO.

----

As a reference, currently supported by alops
https://github.com/HodorNV/ALOps/issues/609

**Comment by freddydk** [comment]
We will add support for this in AL-Go as well

**Comment by fvet** [comment]
@freddydk How is the backlog of AL-GO managed? Based on community votes, does MS prioritize, ...? Can we track progress via github milestones or will the issue appear on the '[Roadmap](https://github.com/orgs/microsoft/projects/521/views/1)'?

**Context**
We're in the process of evaluating the effort of moving away from a customized Azure DevOps config (with a fully functioning pipeline setup) and evaluating /estimating the gaps of implementing standard AL-Go instead. Having some insights on the AL-GO backlog process / ETA of reported issues would be great to set realistic goals internally.

**Comment by freddydk** [comment]
The Roadmap will be the place to look for enhancements and the roadmap (working on that)
Issues created here might be transferred to the roadmap if they are of a certain size - or fixed as issues if they are small. This particular issue is small and will be fixed shortly.

Contributions to AL-Go are also welcome from partners - basically fork the AL-Go repo, create a branch and use Update AL-Go System Files with fork/AL-Go@branch to run on your own code.

Problem is that this small issue requires a fix in ContainerHelper AND in AL-Go - but quite small, but it makes it a little harder, but not impossible, let me show how this is done right here...

This branch on BcContainerHelper contains the code to add this parameter to Run-AlPipeline (which is the function used in AL-Go): https://github.com/freddydk/navcontainerhelper/tree/EnableExternalRulesets (and this is the PR for merging into BcContainerHelper https://github.com/microsoft/navcontainerhelper/pull/3198)

This branch on AL-Go contains the code to add this setting in AL-Go and transfer it to Run-AlPipeline: https://github.com/freddydk/AL-Go/tree/EnableExternalRulesets (draft PR here: https://github.com/microsoft/AL-Go/pull/733)

So... - if you want to try it out, you can use Update AL-Go System Files with freddydk/AL-Go@EnableExternalRulesets - then it should work... (haven't tested it myself yet)

**Comment by freddydk** [comment]
This is now supported in the preview version of AL-Go for GitHub

**Comment by freddydk** [comment]
Shipped in v4.0
